Update to site

tmheunis opened the pull request:
- The legal notice ('impressum') has been updated to reflect github pages hosting and a link to related github privacy terms.
- The main page now includes a link to the support channel for JTrack (closes https://github.com/abcd-j/website/issues/2).
- The main page also includes a section on julearn (closes https://github.com/abcd-j/website/issues/3)

tmheunis committed:
legal notice (impressum) updated

tmheunis committed:
support link for JTrack added to main page

tmheunis committed:
add julearn content and logo to main page

adswa commented on layouts/partials/index_abcdj.html:
```diff
@@ -179,7 +187,41 @@ <h3><a href="https://github.com/Biomarker-Development-at-INM7">GitHub</a></h3>
                     <img src="../pics/jtrack_logo.svg" alt="">
                 </div>
             </div>
+            <a href="#section5" class="profile-scroll push"><i class="fas fa-chevron-down"></i></a>
```
FTR, there also is a support channel for Julearn: https://matrix.to/#/#ml:inm7.de

jsheunis committed:
override theme layout 'single' and exlcude tranlsation list from single pages

jsheunis committed:
updates functionality for language switching

This commit includes a few changes to the header template to:
- display all available languages, not only the translations of the current language
- construct the correct url for the translation of the current page
- Underline the currently selected language
The languages are also displayed in full, instead of using the prefix
